### DISCUSSION TITLE
Change the name

### DISCUSSION THREAD

**Discussion by lucianghinda** [ShortRuby/rubyandrails.info#45] [General]
I am thinking to change the name of the website.

Current: 
- `rubyandrails.info`

Reasons:
- It is too close to `rubyonrails` and it might create confusions
- It has focus on Rails and somehow the majority of resources are about Rails but I would like to think that this is about Ruby in general

Proposal
- `chooseruby.com` 

Downsides:
- Maybe this change will affect SEO and Google indexing. TODO: Find resources about migrating between domains

**Comment by adrianthedev** [comment]
Yes. I think that's a good idea!
I found it confusing as well. I still write rubyonrails.info sometimes when I reference it.

Regarding the migration, I suggest writing a middleware that takes the path the user lands on and redirects with 301 to shortruby.com
I think this is the best way to save the SEO juices.

**Reply by lucianghinda** [reply]
@adrianthedev thank you for the reply. 

After some more thoughts I will move it to `chooseruby.com`
